libs/ui/elements/src/lib/metadata-info/metadata-info.component.ts

```diff
@@ -90,6 +90,12 @@ export class MetadataInfoComponent {
     return this.metadata.contactsForResource?.[0]
   }
 
+  get metadataLandingPageAdvanced() {
```

Comment:
could you add a `// Geocat specific` comment here? So that we keep track of changes :) thanks!